x-pack/legacy/plugins/lens/public/editor_frame_service/editor_frame/_config_panel_wrapper.scss

```diff
@@ -0,0 +1,49 @@
+.lnsConfigPanel__panel {
+  margin-bottom: $euiSizeS;
+}
+
+.lnsConfigPanel__row {
+  background: $euiColorLightestShade;
+  padding: $euiSizeS;
+  border-radius: $euiBorderRadius;
+
+  // Add margin to the top of the next same panel
+  & + & {
+    margin-top: $euiSizeS;
+  }
+}
+
+.lnsConfigPanel__addLayerBtn {
+  color: transparentize($euiColorMediumShade, .3);
+  // sass-lint:disable-block no-important
```

Comment:
I think we should try to keep the same rule as for eslinting disable comments and should leave a comment why it's okay to break with the linting rule here, i.e. I would suggest we leave which `box-shadow` we need to overwrite here.

Reply:
```suggestion
  // Remove EuiButton's default shadow to make button more subtle
  // sass-lint:disable-block no-important
```

Reply:
Done